.github/dependabot.yaml

```diff
@@ -0,0 +1,58 @@
+version: 2
+updates:
+  #upgrades production dependencies of GitHub Actions
```

Comment:
Nit (multiple instances): Prefer imperative tense (e.g., "Upgrade") and a space after comment symbols for readability.